website/common/script/ops/scoreTask.js

```diff
@@ -277,22 +278,38 @@ module.exports = function scoreTask (options = {}, req = {}) {
         task.completed = true;
 
         // Save history entry for daily
+        // If this is a yesterDaily being scored, set the date to the last minute of the user's "yesterday"
+        let dateScored = task.yesterDailyScored ? startOfDay(user.preferences).subtract(1, 'minute') : moment();
```

Comment:
@MeanderingCode If I understand this correctly, `task.yesterDailyScored` is true if and only if the Daily is being marked complete in the "Check off any Dailies you did yesterday" screen - is that right? 

Also, the code later under the comment `// Set whether we're scoring a yesterdaily` -- that code is executed if the user is completing the Daily in the "Check off any Dailies you did yesterday" screen, right?

Reply:
@Alys Yes. I purposefully did not as the property to the model, so it will never be persisted. It is set in the browser for all (loaded) tasks collected by the yesterdailies modal, then the new optional route segment is added to the score API endpoint. On the server, the property is set on the loaded task is the optional segment is included, and is thus available to all functions where the task is passed as a parameter (and not reloaded from the DB).

Reply:
@MeanderingCode Cool, that all sounds good!

I think the "yesterdaily" terminology makes that a little confusing though (not your fault). A Daily's `yesterDaily` attribute is true to indicate that the Daily should appear in the "Check off any Dailies you did yesterday" screen if it was due but not complete, so strictly speaking "a yesterDaily" would be any Daily that's **capable** of appearing in that screen, not necessarily one that is currently being marked complete in that screen. Because of that, when I first looked through your new `yesterDailyScored` code, I thought it was being applied to all such Dailies (which would have been a bug) but I see now that it's applied only to those being completed in the screen (which is good). Maybe the comments could be adjusted slightly to prevent other devs having the same initial misconception I had?

For example, this:
	// If this is a yesterDaily being scored, set the date to the last minute of the user's "yesterday"
could be adjusted to something like this:
	// If this is a yesterDaily being scored in the RYA screen, set the date to the last minute of the user's "yesterday"

("RYA" or "Record Yesterday's Activity" is the official name of the "Check off any Dailies you did yesterday" feature, although unfortunately the name doesn't appear anywhere on the website any more.)

Also this:
	// Set whether we're scoring a yesterdaily
could perhaps become something like this:
	// Set whether we're scoring a yesterdaily in the RYA screen.

And:
// Insert history at the right point, since this could be a yesterdaily scoring after someone else's today scoring
could be:
// Insert history at the right point, since this could be a yesterdaily scored in someone's RYA screen after someone else's today scoring

This is all assuming I'm understanding it correctly now. :) How do you feel about that? If you think my suggested changes to the comments aren't correct, you should definitely say so!

Reply:
Updated comments for clarity. Thanks for the suggestion!